compiler_opt/rl/trainer.py

```diff
@@ -51,7 +51,8 @@ def __init__(
       checkpoint_interval=10000,
       log_interval=100,
       summary_interval=1000,
-      summaries_flush_secs=10):
+      summaries_flush_secs=10,
+      tensorboard_data_interval=100):
```

Comment:
looks a good idea to set it to be 100 here, let's just make it as default

Reply:
and call it summary_log_interval to be consistent and move it between log_interval and summary_interval

also add it to Args in docstring below

Reply:
no need to add a flag cause users can set it with gin config if they really want to change it

Reply:
not needed cause we don't really expect to change these values